storage/innobase/xtrabackup/test/inc/common.sh

```diff
@@ -1150,5 +1150,33 @@ function backup_and_restore() {
   verify_db_state $db
 }
 
+### kill all running queries accept the condition argument
+## example kill_condition_queries "time > 0 & INFO like '%SLEEP%'"
+function kill_condition_queries()
```

Comment:
rename to kill_query_pattern() , you can tell the pattern is passed as argument or WHERE condition

Reply:
fixed